libimage/manifests/manifests.go

```diff
@@ -353,9 +353,12 @@ func (l *list) Add(ctx context.Context, sys *types.SystemContext, ref types.Imag
 			}
 			if instanceInfo.OS == "" {
 				instanceInfo.OS = config.OS
+				instanceInfo.OSVersion = config.OSVersion
+				instanceInfo.OSFeatures = append([]string{}, config.OSFeatures...)
```

Comment:
Can't you directly set instanceInfo.OSFeatures=config.OSFeatures?

Reply:
Sure, it's probably a safe bet that the contents of `config.OSFeatures` won't get changed elsewhere.